ui/pages/home/home.component.js

```diff
@@ -620,6 +629,10 @@ export default class Home extends PureComponent {
       showWhatsNewPopup &&
       !process.env.IN_TEST &&
       !newNetworkAddedConfigurationId;
+
+    const showTermsOfUse =
+      completedOnboarding && !onboardedInThisUISession && showTermsOfUsePopup;
```

Comment:
Does adding `!process.env.IN_TEST` fix the unit test issues?

Reply:
Thanks, this would help with the popover issues.
I think I'll be able to use this to help with the onboarding flow checkbox too.